post_start/script/set_default_graf_dash.sh

```diff
@@ -1,12 +1,20 @@
 #!/bin/sh
 
+# Grafana connection configuration
+GRAFANA_USER=${GRAFANA_USER:-admin}
+GRAFANA_PASS=${GRAFANA_PASS:-admin}
+GRAFANA_HOST=${GRAFANA_HOST:-grafana}
+GRAFANA_PORT=${GRAFANA_PORT:-3000}
+GRAFANA_URI=${GRAFANA_USER}:${GRAFANA_PASS}@${GRAFANA_HOST}:${GRAFANA_PORT}
+
+
 echo "Setting Grafana default dashboard..."
 DASH_UID="sJUFc-NWk"
 DASH_ID=0
 for i in 1 2 3 4 5; do
-    curl -H 'Content-Type: application/json' -X GET http://admin:admin@grafana:3000/api/dashboards/uid/$DASH_UID && RESP=$(curl -H 'Content-Type: application/json' -X GET http://admin:admin@grafana:3000/api/dashboards/uid/$DASH_UID) && DASH_ID=$( echo "$RESP" | jq '.dashboard.id' ) && break || sleep 15;
+    curl -H 'Content-Type: application/json' -X GET http://${GRAFANA_URI}/api/dashboards/uid/$DASH_UID && RESP=$(curl -H 'Content-Type: application/json' -X GET http://${GRAFANA_URI}/api/dashboards/uid/$DASH_UID) && DASH_ID=$( echo "$RESP" | jq '.dashboard.id' ) && break || sleep 15;
```

Comment:
Should we put the full URL into double quotes to make sure white space and other special characters don't break the command?

Reply:
Still working on this. 
Need to debug the quotes, maybe add escape chars somewhere.